src/lib/addClipMarkup.js

```diff
@@ -0,0 +1,100 @@
+/**
+ * Add clip-related markup to page head for Google SEO.
+ *
+ * @param {player} Player An instance of the Player.
+ * @return {void}
+ */
+// TODO: NEED TO DETERMINE WHERE THIS METHOD SHOULD BE DEFINED, IF NOT HERE
+export function addClipMarkup(player) {
+    if (!player) {
+        return;
+    }
+
+    // Any off-site page may have multiple embedded Vimeo videos. We can inject
+    // VideoObject markup for each video. According to the Google SEO team, we
+    // should include 'chapters' data for only one video in the page
+    const scriptElem = document.querySelector("script[type='application/ld+json']");
+    let existingMicrodataHasChapters;
+    let existingMicrodata = [];
+
+    if (scriptElem) {
+        const temp = JSON.parse(scriptElem.textContent);
+        existingMicrodata = Array.isArray(temp) ? temp.slice() : new Array(temp);
+        existingMicrodataHasChapters = existingMicrodata.some((item) => item.hasOwnProperty('hasPart'));
+    }
+
+    player.get('videoObjectMetadata')
```

Comment:
is this already implemented or it's something we'll have to add?

Reply:
@jdreetz are you asking whether the videoObjectMetadata get method is implemented? If so, we're working on that concurrently in the player frontend repo. [Here's a draft branch](https://github.vimeows.com/player/player/compare/test-branch-shawnj-key-moments-ptl-512#diff-648e60e80b0b788ffab0a162ded9fdf85935083cd8210d0c91a997dee9787c8eR600). _That_ change is dependent on an update to player-backend. In retrospect it might have been better to request the reviews in order, from backend to frontend to player.js.

Reply:
@sjcpu4096 yes, I wasn't sure whether that property was exposed yet, but sounds like ya'll are working on it.